react-search/my-app/src/SearchBar.tsx

```diff
@@ -0,0 +1,22 @@
+import { ChangeEvent } from 'react';
+
+type Props = {
+  handleChange: (e: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
This works, how you've done it. However, I don't find it to be very elegant code. I find it nicer to hide the event from the parent and instead extra the value here, in the SearchBar. That way, if I later change my mind and decide to use something besides an Input element, I will only have to change this implementation. If I keep it with the Event like you've done, then I'd also have to change the parent.